server/index.js

```diff
@@ -4,33 +4,48 @@ const app = express();
 const port = 3000;
 const path = require('path');
 const bodyParser = require('body-parser');
-const db = require('../database/index.js');
+
+const mongo = require('../database/mongo.index.js')
 
 app.use(bodyParser.json());
 
 
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use('/books/:id', express.static(path.join(__dirname, '../public')));
 
-app.get('/books/:id/authors/title', async(req, res) => {
-  console.log('server/index line 16')
-  const title = await db.getBook(req.params.id);
-  res.json(title);
+//MONGO VERSION
+
+app.get('/books/:id/authors/title', async (req, res) => {
+    console.log('server/index line 16')
+  var booksId = req.params.id
+  var book =  await mongo.getBooks(booksId)
```

Comment:
Having Mongo code in your `server/index.js` seems suboptimal. When thinking about Separation of Concerns (SOC), I ask myself: "if I change XXX, what files will I have to modify?"

If I change my db choice from MongoDB to YYY, what files will I have to modify?

Does it make sense for me to have to change my `server/index.js`? ...probaby not

How can we organize our code such that changing dbs doesn't force us to refactor our `server/index.js` over and over? Better yet, how can we avoid having multiple versions of this file just for different db choices?